### ISSUE TITLE
We should load a few years of data rather than less than a year

### ISSUE THREAD

**Issue by bjedrzejewski** [ScottLogic/StockFlux#213]
This is quite disappointing for some users :)

**Comment by owennw** [comment]
I'm not sure what this means..

**Comment by bjedrzejewski** [comment]
When you look at the stock data, we are loading less than a year for each stock. It would be good to be able to either load more by default, or select options like - last year, 3 years, 5 years, All data.

**Comment by jleft** [comment]
API addition made in BitFlux's `develop` branch (not released yet). https://github.com/ScottLogic/BitFlux/pull/660

Example:

```js
// Fetch 200 points
bitflux.app().periodsOfDataToFetch(200);

// Fetch 1000 period of data from now
bitflux.app().periodsOfDataToFetch(1000);
```

N.B. for data sources which skip weekends, the number of data points may be less than that specified, as the start date is calculated to be `periodsOfDataToFetch` days before the end date - which includes weekends. Roughly looks like this:

```js
var endDate = new Date(); // today
var startDate = endDate - (periodsOfDataToFetch * day);
```

---

We're looking to update the API to allow the start and end date to be specified in the future - https://github.com/ScottLogic/BitFlux/issues/658 - (possibly in time for the initial public release), but I'm hoping this change is adequate for now?

**Comment by owennw** [comment]
Closed by #466 

Thanks @jleft, that should work a treat.
